src/server/src/main/java/io/cassandrareaper/ReaperApplicationConfiguration.java

```diff
@@ -150,6 +150,9 @@
   @JsonProperty
   private Optional<String> enforcedLocalDatacenter = Optional.empty();
 
+  @JsonProperty
+  private Optional<Boolean> enableConcurrentMigrations = Optional.of(false);
```

Comment:
why is this an `Optional`? (i would have thought `false` is the safe default whenever it is unspecified?)

Reply:
Null is never exposed, it's only an internal state (which indicates the default value).
Because it is never expose I don't think using and exposing Optional is correct.

(The `@Default` annotation is just for docs is my understanding, and it is up to us to implement it to match the same default value.)